src/openloco/station.cpp

```diff
@@ -34,6 +34,11 @@ namespace openloco
 
     // 0x00492640
```

Comment:
Should probably be moved to `station::update_cargo_acceptance`?

Reply:
I've added it to both as they are both correct

Reply:
Oh wait they are actually different.